src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3092,6 +3092,7 @@ void ApplicationManagerImpl::UnregisterApplication(
   MessageHelper::SendOnAppUnregNotificationToHMI(
       app_to_remove, is_unexpected_disconnect, *this);
   request_ctrl_.terminateAppRequests(app_id);
+  policy_handler_->ForceRetrySequenceStop();
```

Comment:
@IGapchuk Why it is here? If 1 app is unregistered- this does not mean that we did not have another one for PTU processing...

Reply:
@AByzhynar Was added checking of a count of registered applications. If has unregistered the last application, it will be run this function.